e2e/__tests__/__snapshots__/each.test.js.snap

```diff
@@ -29,7 +29,16 @@ exports[`shows error message when not enough arguments are supplied to tests 1`]
 
     Missing 1 arguments
 
-      at packages/jest-each/build/bind.js:81:17
+       6 |  */
+       7 | 
+    >  8 | it.each\`
+         | ^
+       9 |   left    | right
+      10 |   \${true} | \${true}
+      11 |   \${true} |
+
+      at packages/jest-each/build/bind.js:80:23
```

Comment:
Would be lovely if we could strip this internal call

Reply:
That would be awesome!

Reply:
it's 2 or 3 layers up, so a bit painful (we'd have to create the error in `index` and pass it into `bind`)

Reply:
HAH, I lied. Pushed 🙂